.github/workflows/build-reusable.yaml

```diff
@@ -100,19 +148,10 @@ jobs:
             --show-version --batch-mode --errors --no-transfer-progress \
             site
 
-      # We upload tests results if the build fails.
```

Comment:
Why is this removed?

Reply:
The test results are no longer necessary, since Develocity provides the same data in a more browseable format.

I restored this step partially in https://github.com/apache/logging-parent/pull/229/commits/bbd733b135586889bee24b62a4510f52e752c7f7, since Develocity does not upload the test log files generated under `target/logs`.